l3kernel/l3token.dtx

```diff
@@ -649,6 +649,43 @@
 %   for an undefined token.
 % \end{function}
 %
+% \begin{function}[EXP, pTF, added = 2025-05-12]{\token_if_control_symbol:N}
+%   \begin{syntax}
+%     \cs{token_if_control_symbol_p:N} \meta{token}
+%     \cs{token_if_control_symbol:NTF} \meta{token} \Arg{true code} \Arg{false code}
+%   \end{syntax}
+%   Test if the \meta{token} consists of the escape character followed by
```

Comment:
```suggestion
%   Tests if the \meta{token} consists of the escape character followed by
```

Reply:
"Tests whether . . . " is probably better English here!